apps/callscreen/js/call_screen.js

```diff
@@ -372,14 +372,15 @@ var CallScreen = {
   },
 
   showClock: function cs_showClock(now) {
-    LazyL10n.get(function localized(_) {
+    navigator.mozL10n.ready(function() {
       var f = new navigator.mozL10n.DateTimeFormat();
-      var timeFormat = window.navigator.mozHour12 ? _('shortTimeFormat12') :
-                                                    _('shortTimeFormat24');
+      var timeFormat =
+        window.navigator.mozHour12 ? navigator.mozL10n.get('shortTimeFormat12')
+                                   : navigator.mozL10n.get('shortTimeFormat24');
       // FIXME/bug 1060333: Replace span with hidden mechanism.
       // Don't show am/pm (for 12 or 24 time) in the callscreen
       timeFormat = timeFormat.replace('%p', '<span>%p</span>');
-      var dateFormat = _('longDateFormat');
+      var dateFormat = navigator.mozL10n.get('longDateFormat');
```

Comment:
if you want to use the opportunity to remove sync .get, you can use mozL10n.formatValue here. This will be compatible with the upcoming v3 API then :)